tshirts.js

```diff
@@ -1,16 +1,16 @@
-const {expect} = require('chai');
+const { expect } = require("chai");
 
 function size(cms) {
-    if (cms < 38) {
-        return 'S';
-    } else if (cms > 38 && cms < 42) {
-        return 'M';
-    } else {
-        return 'L';
-    }
+  if (cms < 38) {
+    return "S";
+  } else if (cms > 38 && cms < 42) {
+    return "M";
+  } else {
+    return "L";
+  }
 }
-
-expect(size(37)).equals('S');
-expect(size(40)).equals('M');
-expect(size(43)).equals('L');
-console.log('All is well (maybe!)');
+expect(size(37)).equals("S");
+expect(size(38)).equals("S");
+expect(size(40)).equals("M");

```

Comment:
42 is a boundary value. Test cases are important for covering boundary values too.